docs/concepts/mirascope_cli.md

```diff
@@ -99,10 +99,10 @@ from mirascope.prompts import Prompt, messages
 @messages
 class BookRecommendationPrompt(Prompt):
     """
-		SYSTEM:
-		You are an expert in your field giving advice
+	SYSTEM:
```

Comment:
these still look too indented?

Reply:
<img width="759" alt="image" src="https://github.com/Mirascope/mirascope/assets/15950811/70eb8409-b763-4054-adfb-8b423f85ce8f">

Reply:
i think github is making it look weird. maybe pull branch and check on editor and see if its reasonable

Reply:
Yeah seem fine in my editor. Weird.